packaging/macOS/my_hooks/hook-pendulum.py

```diff
@@ -0,0 +1,22 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
```

Comment:
I think we can remove this license given the code  is really trivial and we modified it